data/dapis/avalanche.json

```diff
@@ -1,5 +1,4 @@
 {
-  "AVAX/USD": "0x098cf42f8ea393f32ec9b788d48eb58d044a51a8793f1d98ceb27d25fccbeb2b",
-  "BTC/USD": "0x0a6d78c904e255189a08bfd0773f6fc9da2c5a02a65aa16c8ec83dd346825fe4",
```

Comment:
This one is being referenced in this beaconSet but I don't think the beaconSet is being used anyways: https://github.com/api3dao/operations/blob/update-dapis/data/explorer/beaconSets.json#L3-L4

Reply:
Yeah no worries about this I suppose